examples/editor/src/main.tsx

```diff
@@ -1,14 +1,24 @@
-import React from "react";
-import ReactDOM from "react-dom";
-import App from "./App";
 import "./index.css";
+import styles from "./App.module.css";
+import { mountBlockNoteEditor } from "../../../packages/core/src/BlockNoteEditor";
```

Comment:
imports should reference @blocknote/core or @blocknote/react.

Reply:
I might need your help with @blocknote/react since AFAIK it has to be published to npm, but I can change the local imports from @blocknote/core to npm imports now.